src/pages/Encounters/EncounterShow.tsx

```diff
@@ -249,7 +249,7 @@ export const EncounterShow = (props: Props) => {
             }
             className="btn btn-primary m-1 w-full hover:text-white"
           >
-            Discussion Notes
+            {t("discussion_notes")}
```

Comment:
remove this button, we no longer have this in new architecture. The Notes tab in encounter page is the new thing.